scylla-cdc-base/src/main/java/com/scylladb/cdc/model/worker/Worker.java

```diff
@@ -100,13 +129,25 @@ private TaskActionsQueue queueFirstActionForEachTask(Map<TaskId, SortedSet<Strea
      * At each iteration, runs a single action from |actions| queue if any
      * available.
      */
-    private void performActionsUntilStopRequested(TaskActionsQueue actions) {
-        while (!workerConfiguration.transport.shouldStop()) {
-            try {
-                actions.runNextAction();
-            } catch (InterruptedException e) {
-                // Ignore InterruptedException
-            }
+    @SuppressWarnings("deprecation")
```

Comment:
Why are you deprecating this method? There should at least be a comment saying what's the preferred alternative.

Reply:
Read the annotation. I am _using_ a deprecated method and want to avoid the warning.

Reply:
Right. My bad. I misread the annotation.